### PR TITLE
upgrade: Update admin and nodes repo names

### PR DESCRIPTION
**Pull request by jgravel**
The repo names still point to SOC6. Update them to point to SOC7.

### CODE REVIEW

File: assets/app/features/upgrade/controllers/upgrade-nodes-repositories-checks.controller.js [hunk 1]
```diff
@@ -29,45 +29,45 @@
...
+                        label: 'upgrade.repositories.codes.SLE12-SP3-HA-Pool'
                     },
-                    'SLE12-SP2-HA-Updates': {
+                    'SLE12-SP3-HA-Updates': {
                         status: false,
-                        label: 'upgrade.repositories.codes.SLE12-SP2-HA-Updates'
+                        label: 'upgrade.repositories.codes.SLE12-SP3-HA-Updates'
                     },
                 },
                 'ceph': {
```
Comment (jsuchome) [inline assets/app/features/upgrade/controllers/upgrade-nodes-repositories-checks.controller.js:41]: I think we can remove the `ceph` part completely for 7-8, as we're gonna remove handling of ceph nodes form the upgrade

Comment (jsuchome) [reply assets/app/features/upgrade/controllers/upgrade-nodes-repositories-checks.controller.js:41]: That applies for all places showing `SUSE-Enterprise-Storage` strings (mostly repo labels I believe)

**Comment by jsuchome** [comment]
I'm not yet sure if we should be doing this against master branch. Or maybe the branch is correct, but first we must make sure what's the branch for 6-7 upgrade... ?

File: assets/app/features/upgrade/controllers/upgrade-administration-repositories-checks.controller.js [hunk 2]
```diff
@@ -26,21 +26,21 @@
...
+                    label: 'upgrade.repositories.codes.SLES12-SP3-Pool'
                 },
-                'SLES12-SP2-Updates': {
+                'SLES12-SP3-Updates': {
                     status: false,
-                    label: 'upgrade.repositories.codes.SLES12-SP2-Updates'
+                    label: 'upgrade.repositories.codes.SLES12-SP3-Updates'
                 },
-                'SUSE-OpenStack-Cloud-7-Pool': {
+                'SUSE-OpenStack-Cloud-8-Pool': {
```
Comment (jsuchome) [inline assets/app/features/upgrade/controllers/upgrade-administration-repositories-checks.controller.js]: I believe this should be **SUSE-OpenStack-Cloud-Crowbar-8-Pool** .. but it's possible API is not updated

File: assets/app/features/upgrade/controllers/upgrade-nodes-repositories-checks.controller.js (hunk 1 above)
Comment (skazi0) [reply assets/app/features/upgrade/controllers/upgrade-nodes-repositories-checks.controller.js]: https://github.com/crowbar/crowbar-ui/pull/175

Comment (skazi0) [reply assets/app/features/upgrade/controllers/upgrade-nodes-repositories-checks.controller.js:41]: The removal of ceph related code should be covered by https://github.com/crowbar/crowbar-ui/pull/175

**Comment by skazi0** [comment]
@jsuchome regarding branching, my changes to packages were accepted in Staging so they should become official soon. After that we can start merging changes to crowbar-ui/master. Then this change would belong there.

**Review by jsuchome — APPROVED** [review]
LGTM

**Comment by skazi0** [comment]
I'm not sure if it was my (outdated) setup but on Cloud7 side I still got OpenStack repos without "Crowbar" in name. For nodes repos (Cloud8) side it was all fine. More testing ongoing...

**Comment by jsuchome** [comment]
@skazi0 do you have https://github.com/crowbar/crowbar-core/pull/1596 ?

**Comment by skazi0** [comment]
@jsuchome not sure, I'm rebuilding the cloud already.

**Review by skazi0 — APPROVED** [review]
tested and working

**Pull request merged**